README.md

```diff
@@ -24,143 +19,7 @@ Bench is a command-line utility that helps you to install, update, and manage mu
 
 ## Installation
 
-A typical bench setup provides two types of environments &mdash; Development and Production.
-
-The setup for each of these installations can be achieved in multiple ways:
-
- - [Docker Installation](#docker-installation)
- - [Easy Install Script](#easy-install-script)
- - [Manual Installation](#manual-installation)
-
-We recommend using either the Docker Installation or the Easy Install Script to setup a Production Environment. For Development, you may choose either of the three methods to setup an instance.
-
-Otherwise, if you are looking to evaluate ERPNext, you can also download the [Virtual Machine Image](https://erpnext.com/download) or register for [a free trial on erpnext.com](https://erpnext.com/pricing).
-
-
-### Docker Installation
-
-A Frappe/ERPNext instance can be setup and replicated easily using [Docker](https://docker.com). The officially supported Docker installation can be used to setup either of both Development and Production environments.
```

Comment:
Only the docker setup may be removed/linked to the frappe_docker repo. Everything else can stay here as is.